src/PostgREST/Auth.hs

```diff
@@ -71,8 +70,8 @@ jwtClaims secret input time =
   value2map _          = M.empty
 
 {-| Receives the name of a role and returns a SET ROLE statement -}
-setRole :: Text -> BS.ByteString
-setRole r = "set local role " <> cs (pgFmtLit r) <> ";"
+setRole :: Value -> BS.ByteString
```

Comment:
I'm not pleased with this setRole function, I feel it could be rolled into claimsToSQL if the claims were properly prepared (e.g. non-role claims would get prepended by `postgrest.claims.` and all claims could be unioned with the singleton map {role: anonymous} to create a default value for the role).